.github/actions/setup/action.yml

```diff
@@ -5,13 +5,13 @@ runs:
   steps:
     # https://pnpm.io/continuous-integration
     - name: Install package manager
-      uses: pnpm/action-setup@v2
+      uses: pnpm/action-setup@v3
```

Comment:
Interesting, I thought Renovate was catching these things, might need to double check that set-up again :-)

Reply:
Thought so too :D